cloud_foundry_api/assets/configuration/spec.yaml

```
@@ -0,0 +1,76 @@
+name: Cloud Foundry API
+files:
+- name: cloud_foundry_api.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/default
    - template: instances/http
```

You may want to move both template to the end of the section as to not drown out the important config options specific to this integration